.claude/agents/test-writer.md

```
@@ -0,0 +1,52 @@
+---
+name: test-writer
+description: Writes pytest tests for climate_indices functions using TDD. Invoked before implementation code is written.
+---
+
+You are a TDD specialist for the `climate_indices` scientific Python library.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)